src/components/Header/Timer.jsx

```diff
@@ -0,0 +1,90 @@
+import React, { useState, useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux'
+import { startTimer, pauseTimer, stopTimer } from '../../actions/timer';
+import {
+  Badge,
+  Button,
+} from 'reactstrap'
+import TimeEntryForm from '../Timelog/TimeEntryForm'
+import './Timer.css'
+
+const Timer = () => {
+  const userId = useSelector(state => state.auth.user.userid);
+  const pausedAt = useSelector(state => state.timer.seconds);
+  const dispatch = useDispatch();
+
+  const [seconds, setSeconds] = useState(pausedAt);
+  const [isActive, setIsActive] = useState(false);
+  const [modal, setModal] = useState(false);
+
+  const toggle = () => setModal(modal => !modal);
+
+  const reset = () => {
+    setSeconds(0);
+    setIsActive(false);
+  }
+
+  const handleStart = async event => {
```

Comment:
I didn't know the existence of this coding conventions document. Is this a new one? Sorry, I might miss some of your discussion because I was actually half inactive in the team since March. I didn't use either ESlint or VS Code configuration. Would you like to provide more detailed instructions on it? If we are going to have a consistent coding style for the whole project, should we have the same configuration files for the whole team?

Reply:
thats exactly what i'm working on, creating the instructions on how to implement that i think those conventions are kind of new. I think there are some configuration files included on the git but apparently they need some manual installation of ESLint extension to work on VS code, once i have the process on how to do nailed down i will share it with everyone